tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -33,13 +34,30 @@ def register_agent():
     yield
     host_manager.get_host(agent_host).ansible('command', f'service wazuh-agent restart', check=False)
 
+    # Wait until the agent is reconnected
+    timeout = time() + 180
```

Comment:
Please use variables instead of hardcoded timeout.
```suggestion
    time_to_sync = 10
    time_to_agent_reconnect = 180
    ...
    timeout = time() + agent_reconnect_timeout
```